package.json

```diff
@@ -19,13 +19,14 @@
     "generateLocales": "node ./scripts/generateLocales",
     "test": "yarn generateLocales && BABEL_ENV=test jest -i --detectOpenHandles --forceExit",
     "typings": "tsc --p ./tsconfig.build.json",
-    "build": "yarn generateLocales && rollup-all",
+    "clean": "rimraf ./build",
+    "build": "yarn clean %% yarn generateLocales && rollup-all",
```

Comment:
there is a mistake here, %% instead of &&

Reply:
we can also remove the typings command since rollup-all generates typescript definitions too.

Reply:
I uploaded the changes. I fixed the mistake and removed typings command.